website/docs/language/resources/provisioners/local-exec.mdx

```diff
@@ -57,6 +57,12 @@ The following arguments are supported:
 * `environment` - (Optional) block of key value pairs representing the
   environment of the executed command. inherits the current process environment.
 
+* `when` - (Optional) If provided, specifies the timing when the command will
+  be executed. If `when = destroy` is specified, the provisioner will run when
+  the resource it is defined within is destroyed. See
+  [Destroy-Time Provisioners](/language/resources/provisioners/syntax#destroy-time-provisioners)
+  for more details.
```

Comment:
Just a nit to make the language more inclusive and remove a few extra words. Thank you!

```suggestion
* `when` - (Optional) If provided, specifies Terraform will execute the command. For example, `when = destroy` specifies that the provisioner will run when
  the associated resource is destroyed. Refer to
  [Destroy-Time Provisioners](/language/resources/provisioners/syntax#destroy-time-provisioners)
  for details.
```

Reply:
I got it. Thank you for your review. I'll fix it.